seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestConstant.java

```diff
@@ -44,6 +44,11 @@ public class RestConstant {
     public static final String ERROR_MSG = "errorMsg";
 
     public static final String METRICS = "metrics";
+
+    public static final String LOG_PATH = "seatunnel.logs.path";
+
+    public static final String LOG_FILE_NAME = "seatunnel.logs.file_name";
```

Comment:
Please add a comment indicating that this configuration is associated with a shell script